app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -210,6 +215,7 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 			migration(toVersion = 5) {
 				// Migrate to audio behavior enum
 				putEnum("audio_behavior", if (it.getString("pref_audio_option", "0") == "1") AudioBehavior.DOWNMIX_TO_STEREO else AudioBehavior.DIRECT_STREAM)
+				putBoolean("audio_night_mode", if (it.getString("pref_audio_night_mode", "0") == "1") true else false)
```

Comment:
Migrations are only used to migrate previous preference -> new preference. This is a new preference so no migration is needed.

Reply:
Fixed, thanks!